release_extra.py

```diff
@@ -6,6 +6,7 @@
 
 # get the firmware name from the [firmware] section in platformio.ini
 firmware_name = config.get("firmware", "name")
+firmware_name = firmware_name.replace('\\\"', '')
```

Comment:
I know you mentioned this before, but what is the reason for this? I don't remember the specifics but it was a real pain trying to get the firmware name to pass thru as DEFINEs for use inside the firmware.

I would be very interested if you have found a way to tidy that up!

Reply:
The reason for the escape quotes was this define for LVGL in platformio.ini
`  -DLV_MEM_CUSTOM_INCLUDE=\"esp32-hal-psram.h\"
`
I had to make sure that the quotes are passed thru to the place in LVGL code where the `#include ....` is put together depending on customisation. This was necessary for linux only what is used in the automated release build.
I guess there is a similar requirement for the firmware name and such to be processed in the tool chain and get finally right in your code.
Unfortunately the firmware name keeps the escape quotes and uses ugly paths during build
`Linking .pio/build/wt32-eth/\"OXRS-IO-TouchPanel-ESP32-FW\"_wt32-eth_v2.0.1.elf
`
With the above modification I remove the escaped quotes to have a clean firmware name for further use.